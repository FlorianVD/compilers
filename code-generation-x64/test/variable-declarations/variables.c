void do_print()
{
    int i = 42;
    int j = 100;
    int k;
    k = 7;

    print(i);
    print(j);
    print(k);
    k = 14;
    print(k);
}

int main()
{
    do_print();

    return 0;
}
