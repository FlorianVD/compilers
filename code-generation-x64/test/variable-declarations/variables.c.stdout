42
100
7
14
